examples/resources/configuration/subscriptions.yaml

```diff
@@ -0,0 +1,137 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: subscriptions
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: subscriptions.eventing.kyma-project.io
```

Comment:
Hm, shouldn't `name` and `app.kubernetes.io/name` label be the same? ConfigMap automatically adds that label based on the name. We have a similar approach with other extensibility configurations.